build.rs

```diff
@@ -18,10 +17,10 @@ fn main() {
 
     let seed = env::var("RHAI_AHASH_SEED").map_or_else(|_| "None".into(), |s| format!("Some({s})"));
 
-    contents = contents.replace("{{AHASH_SEED}}", &seed);
+    contents = contents.replace("{{ AHASH_SEED }}", &seed);
```

Comment:
Requiring a space after and before the `{{ ... }}` may make the template brittle...

Reply:
Will revert

Reply:
Unless you want it to auto-format as Rust... in that case the editor may auto-add those spaces...